remote_fun/fun_common.hrl

```diff
@@ -0,0 +1,25 @@
+-define(SERVER, function_server@localhost).
```

Comment:
+1

Reply:
it is pretty strange to run a -noshell flag in an erlang system. you want to see what is happening. you could connect with a remote shell to the system for doing so, but the standard way of doing things is to leave the shell working. that is why you need two makefile targets for these since you need to run two shells.